driver/pace/driver/driver.py

```diff
@@ -239,6 +239,12 @@ def exit_function(*args, **kwargs):
                     exit(0)
 
                 setattr(self, "step_all", exit_function)
+            self.config.stencil_config.dace_config = DaceConfig(
+                communicator=communicator,
+                backend=self.config.stencil_config.backend,
+                tile_nx=self.config.nx_tile,
+                tile_nz=self.config.nz,
+            )
             orchestrate(
                 obj=self,
                 config=self.config.stencil_config.dace_config,
```

Comment:
Outside the scope of this PR, but is there a reason this can't be a decorator on the method, other than that it takes a config that could be statefully built?

Reply:
It's mostly the config. Also, intellectually I liked the idea that you could orchestrated code _outside_ of the object itself (like a standing function orchestrating on-the-fly) though of course you can always call a decorator by hand.